subzero-core/src/main/java/info/jerrinot/subzero/internal/strategy/KryoStrategy.java

```diff
@@ -41,10 +47,10 @@ private Kryo newKryoInstance() {
             kryo = new Kryo();
         } else {
             ClassLoader classLoader = ClassLoaderUtils.getConfiguredClassLoader(hazelcastInstance);
```

Comment:
shouldnt you respect the properties even when the `IGNORE_HAZELCAST_CLASSLOADER` is set to `true` ?

Reply:
That's a good question. We could assume that the property `com.esotericsoftware.kryo.util.MapReferenceResolver` should ignore whether the Hazelcast class loader is ignored or not as we care only about the `ReferenceResolver`.
So in line 42 we should have sth like this:
```
kryo = new Kryo(createReferenceResolver());
```
Do you agree?

Reply:
yes. :point_up: looks good.
thanks for the PR!